### PR TITLE
Fix ntp time offset reset, fix logs output

### PR DESCRIPTION
**Pull request by AndrewBoyarsky**

### CODE REVIEW

File: apl-exec/src/main/resources-dev/logback.xml [hunk 1]
```diff
@@ -44,6 +44,5 @@
     <root level="severe">
```
Comment (yuriy-larin-firstbridge) [inline apl-exec/src/main/resources-dev/logback.xml]: remove incorrect LEVEL in root

File: apl-exec/src/main/resources-prod/logback.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36}:%line - %msg%n</pattern>
         </encoder>
     </appender>
-    <logger name="com.apollocurrency.aplwallet" level="DEBUG" additivity="false"/>
+    <logger name="com.apollocurrency.aplwallet" level="DEBUG"/>
     <!--<logger name="com.apollocurrency.aplwallet.apl.core.peer" level="TRACE"/>-->
     <!--Uncomment next line to enable trim duration logging-->
     <!--<logger name="com.apollocurrency.aplwallet.apl.db.VersionedEntityDbTable" level="trace"/>-->
```
Comment (yuriy-larin-firstbridge) [inline apl-exec/src/main/resources-prod/logback.xml]: remove incorrect LEVEL in root

File: apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java [hunk 3]
```diff
@@ -38,13 +39,16 @@ private void setTimeDrift() {
...
                     + ", clock offset(ms)=" + offset); // offset in ms
-            
+
             timeOffset = offsetValue;
         }
-        catch (IOException e ) {
+        catch (SocketTimeoutException e) {
+            LOG.warn(e.getMessage() + " will keep prev offset " + timeOffset, e);
+        }
+        catch (IOException e) {
```
Comment (yuriy-larin-firstbridge) [inline apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java]: whats about processing UnknownHostException here  ??
> Thrown to indicate that the IP address of a host could not be determined.

File: apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java [hunk 4]
```diff
@@ -58,9 +62,9 @@ public NtpTime() {}
 
     @PostConstruct
     public void start() {
+        setUpClient();
         Runnable timeUpdate = this::setTimeDrift;
         ThreadPool.scheduleThread("NTP Update", timeUpdate, REFRESH_FREQUENCY, TimeUnit.SECONDS);
-        setUpClient();
     }
 
     private void setUpClient() {
```
Comment (yuriy-larin-firstbridge) [inline apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java]: I would recommend catch here two errors (or IO)

... catch (SocketException | UnknownHostException e) {
OR
... catch (IOException  e) { // that replaces both listed above

Comment (AndrewBoyarsky) [reply apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java]: Why we should catch such exceptions?

File: apl-exec/src/main/resources-dev/logback.xml (hunk 1 above)
Comment (AndrewBoyarsky) [reply apl-exec/src/main/resources-dev/logback.xml]: set to warn

File: apl-exec/src/main/resources-prod/logback.xml (hunk 2 above)
Comment (AndrewBoyarsky) [reply apl-exec/src/main/resources-prod/logback.xml]: set to warn

File: apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java (hunk 3 above)
Comment (AndrewBoyarsky) [reply apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/NtpTime.java]: done

**Pull request merged**